### ISSUE TITLE
Play in background/Add to playlist option in the Watched List

### ISSUE THREAD

**Issue by 2SI3NX** [TeamNewPipe/NewPipe#1229]
- [ x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x ] I checked if the issue/feature exists in the latest version.

The option to play in background or adding to playlist is available when searching for a video but it is not available when you're in the Watched list. Besides deleting videos from Watched list when long-pressing a video, it would be nice to have the other options there as well.

**Comment by speednroll** [comment]
Is this not the same as what I requested earlier in #998 ?

**Comment by theScrabi** [comment]
Yes ... Yes makes sense. Also when this is available in the rest of the UI we should add it here as well.

**Comment by 2SI3NX** [comment]
@speednroll yes actually that is the same as what you requested on #998
